### ISSUE TITLE
feature/suepr fit method and convert raw matrix to math.js matrix

### ISSUE THREAD

**Issue by JasonShin** [JasonShin/kalimdorjs#106]
* **I'm submitting a ...**
[/] feature request

* **Summary**
... add more details
-> Work on this once @benjaminmcdonald's PR is merged in

**Comment by JasonShin** [comment]
Won't do
